app/models/concerns/orderable.rb

```diff
@@ -2,9 +2,16 @@ module Orderable
   extend ActiveSupport::Concern
 
   included do
-    default_scope { 
-      order(featured: :desc, comments_count: :desc, created_at: :desc, id: :desc) 
+    scope :order_by, lambda{ |field|
+      if (field)
```

Comment:
Do you need to include an else in this conditional so that they come back in their default scope or will that already happen? Haven't messed around with scopes that much before.